core/src/test/java/ai/timefold/solver/core/impl/score/buildin/BendableBigDecimalScoreDefinitionTest.java

```diff
@@ -96,16 +96,16 @@ void createScore() {
     @Test
     void divideBySanitizedDivisor() {
         BendableBigDecimalScoreDefinition scoreDefinition = new BendableBigDecimalScoreDefinition(1, 1);
-        BendableBigDecimalScore dividend = scoreDefinition.createScoreUninitialized(2, BigDecimal.ZERO, BigDecimal.TEN);
+        BendableBigDecimalScore dividend = scoreDefinition.createScore(BigDecimal.ZERO, BigDecimal.TEN);
```

Comment:
We are modifying several test files that can be simplified by using the `var` pattern. However, the real question is: do we want to apply this pattern to all those files?

Reply:
I went through every single modified file and converted them to `var`.
When re-reviewing, I recommend going commit-by-commit and skipping the last few, which are nothing but hundreds of lines of `var` replacements.